### PR TITLE
Add basic test for duplicate

### PR DESCRIPTION
**Pull request by pwalczysko**
Basic test for the duplicate command. The first iteration just checks whether the original object is still present.

### CODE REVIEW

**Comment by ximenesuk** [comment]
```
OmeroPy colin$ ./setup.py test -t test/integration/clitest/test_duplicate.py -v
...
collected 10 items

test/integration/clitest/test_duplicate.py::TestDuplicate::testDeleteMyData[Image] PASSED
test/integration/clitest/test_duplicate.py::TestDuplicate::testDeleteMyData[Image-I] PASSED
test/integration/clitest/test_duplicate.py::TestDuplicate::testDeleteMyData[Dataset] PASSED
test/integration/clitest/test_duplicate.py::TestDuplicate::testDeleteMyData[Dataset-I] PASSED
test/integration/clitest/test_duplicate.py::TestDuplicate::testDeleteMyData[Project] PASSED
test/integration/clitest/test_duplicate.py::TestDuplicate::testDeleteMyData[Project-I] PASSED
test/integration/clitest/test_duplicate.py::TestDuplicate::testDeleteMyData[Plate] PASSED
test/integration/clitest/test_duplicate.py::TestDuplicate::testDeleteMyData[Plate-I] PASSED
test/integration/clitest/test_duplicate.py::TestDuplicate::testDeleteMyData[Screen] PASSED
test/integration/clitest/test_duplicaty::TestDuplicate::testDeleteMyData[Screen-I] PASSED

================================================ 10 passed in 42.17 seconds =================================================
```

:+1:

File: components/tools/OmeroPy/test/integration/clitest/test_duplicate.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+class TestDuplicate(CLITest):
+
+    def setup_method(self, method):
+        super(TestDuplicate, self).setup_method(method)
+        self.cli.register("duplicate", DuplicateControl, "TEST")
+        self.args += ["duplicate"]
+
+    @pytest.mark.parametrize("model", model)
+    @pytest.mark.parametrize("object_type", object_types)
+    def testDeleteMyData(self, object_type, model):
```
Comment (ximenesuk) [inline components/tools/OmeroPy/test/integration/clitest/test_duplicate.py]: The name of the test may need changing!

**Comment by pwalczysko** [comment]
@ximenesuk pushed a fix, hopefully this is not too long a name

**Comment by ximenesuk** [comment]
```
test/integration/clitest/test_duplicate.py::TestDuplicate::testDuplicateSingleObject[Image] PASSED
test/integration/clitest/test_duplicate.py::TestDuplicate::testDuplicateSingleObject[Image-I] PASSED
test/integration/clitest/test_duplicate.py::TestDuplicate::testDuplicateSingleObject[Dataset] PASSED
test/integration/clitest/test_duplicate.py::TestDuplicate::testDuplicateSingleObject[Dataset-I] PASSED
test/integration/clitest/test_duplicate.py::TestDuplicate::testDuplicateSingleObject[Project] PASSED
test/integration/clitest/test_duplicate.py::TestDuplicate::testDuplicateSingleObject[Project-I] PASSED
test/integration/clitest/test_duplicate.py::TestDuplicate::testDuplicateSingleObject[Plate] PASSED
test/integration/clitest/test_duplicate.py::TestDuplicate::testDuplicateSingleObject[Plate-I] PASSED
test/integration/clitest/test_duplicate.py::TestDuplicate::testDuplicateSingleObject[Screen] PASSED
test/integration/clitest/test_duplicate.py::TestDuplicate::testDuplicateSingleObject[Screen-I] PASSED
test/integration/clitest/test_duplicate.py::TestDuplicate::testDuplicateSingleObjectDryRun PASSED
test/integration/clitest/test_duplicate.py::TestDuplicate::testDuplicateSingleObjectReport PASSED
```
